comparison.md

```diff
@@ -48,5 +47,25 @@ None of these problems are insurmountable. For example PNaCl defines a small por
 * Did I mention Microsoft?
 
 # RISCV
+### Isn't RISCV a purely hardware spec?
+
+# JVM
+### Nonstarter.  Oracle ownership and intellectual property issues.
+
+# EVM1 -> EVM2
+### Good:
+* We own it.
+* It can evolve to where we need to go.
+* Coordination with other projects is optional.
+* We have the engineering talent we need.
+* We can incorporate wasm and other tech as appropriate.
```

Comment:
At the least:
1. A module could be attached to the JIT hook to translate EVM to WASM.
2. An unused EVM opcode could be used to punch through to an extended instruction set, which could be WASM.
The success of first could be a step towards the second.